release-team/role-handbooks/enhancements/README.md

```diff
@@ -101,48 +115,62 @@ It is important that this process be followed and documentation remain up-to-dat
 - Verify issues have k/k PRs associated so they can be referenced and easily tracked. This is going to be critical come Enhancement Freeze and Code Freeze to see the status of the code.
 - Work with the Release Lead to introduce yourself, talk about release information, and relay information about opting into the release with SIG Leads.
 - Weekly Release meetings require updates of current status. Use the `Feature stats` tab to update the release team on counts of enhancements in good and bad progress.
-- Start reminding Issue owners that KEPs are required for each enhancement and that KEPs must be in an implementable state by Enhancement Freeze.
 
 #### Collection Monitoring and Triage (Week 2-3)
 
-- Move data from `KEP Collection` into the `data` tab to be tracked officially for the release
-- Stay on top of comments in issues when owners respond. Apply correct labels, milestone information, update their status in the sheet if necessary.
+- Move data from `KEP Collection` into the `data` tab to be tracked officially for the release.
```

Comment:
I think something like: "Monitor the `KEP Collection` tab regularly to new additions as SIGs enter opted-in KEPs into the release"  just a smidge more explicit about why they are checking that tab

Reply:
Also something like: Move data from `KEP Collection` into the `data` tab to be tracked officially for the release and verify that the KEP is now included in the `Enhancements` tab.

Reply:
done!